patches/chrome-android-BUILD.gn.patch

```diff
@@ -42,31 +42,39 @@ index 32a57e574780970a458c25b76ebc57d7b981d813..cfbe384938395e3013e6c7c3567facc4
    }
  
    action_with_pydeps("chrome_android_java_google_api_keys_srcjar") {
-@@ -1604,6 +1609,7 @@ if (current_toolchain == default_toolchain) {
+@@ -1148,6 +1153,7 @@ if (current_toolchain == default_toolchain) {
+       "//components/segmentation_platform/public:junit",
+       "//components/translate/content/android:junit",
+     ]
++    deps += ["//brave/browser/custom_app_icons/android:custom_app_icons_junit_tests"]
```

Comment:
this does not follow patching standards, please use a variable

Reply:
@bridiver updated the changes here : https://github.com/brave/brave-core/pull/28287/commits/32382203969524126d4035da7ad2f8544415e052